extensions/azure/cosmos/contract-definition-store-cosmos/src/test/java/org/eclipse/dataspaceconnector/contract/definition/store/TestFunctions.java

```diff
@@ -20,7 +34,7 @@ public static ContractDefinition generateDefinition() {
                 .build();
     }
 
-    public static ContractDefinitionDocument generateDocument() {
-        return new ContractDefinitionDocument(generateDefinition());
+    public static ContractDefinitionDocument generateDocument(String partitionKey) {
```

Comment:
we do, but test classes need control over the key, since they also instantiate the `CosmosContractDefinitionStore` and pulling out the `TEST_PARTITION_KEY` from `TestFunctions` feels kind of awkward.